src/modules/test/commentsCount.test.js

```diff
@@ -0,0 +1,22 @@
+import commentCounter from '../commentscount.js';
+
+global.fetch = jest.fn();
+describe('commentCounter', () => {
+  beforeEach(() => {
+    fetch.mockReset();
+  });
+
+  test('returns the number of comments', async () => {
+    const comment = [{ id: 1 }, { id: 2 }, { id: 3 }];
+    fetch.mockResolvedValue({ json: () => Promise.resolve(comment) });
+    const result = await commentCounter();
+    expect(result.length).toBe(3);
+  });
+
+  test('returns the comments object', async () => {
+    const comment = [{ id: 1 }, { id: 2 }, { id: 3 }];
+    fetch.mockResolvedValue({ json: () => Promise.resolve(comment) });
```

Comment:
- [ ]  You are supposed to cover the tests of all your Counter module, you've tested comment and movies counter, but you didn't cover the like counter. please implement the test for like counter as it was suggested in the requirement . This will make your project more professional.

Reply:
We have added the likes counter to the test cases